### PR TITLE
Update tidb-lightning-distributed-import.md

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 1]
```diff
@@ -18,13 +18,11 @@ TiDB Lightning 并行导入可以用于以下场景：
 >
 > - 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
 >
-> - 并行导入一般用于 local-backend 模式。
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-distributed-import.md:21]: local-backend 应该是弃用的术语

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 2]
```diff
@@ -18,13 +18,11 @@ TiDB Lightning 并行导入可以用于以下场景：
...
->
 > - 并行导入一般用于 Physical Import Mode；虽然也能用于 Logical Import Mode，但是一般不会带来明显的性能提升。
 
 ## 使用说明
 
-使用 TiDB Lightning 并行导入无须额外配置。TiDB Lightning 在启动时，会在下游 TiDB 中注册元信息，并自动检测是否有其他实例向目标集群导入数据。如果有，则自动进入并行导入模式。
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-distributed-import.md:27]: 还是需要额外为多个 lightning 分配要导入的文件

**Review by lance6716 — COMMENTED** [review]
/cc @sunzhaoyang

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
The changes made in this pull request were covered in https://github.com/pingcap/docs-cn/pull/12828. The pull request is now being closed.

**Pull request closed without merging**